fiftyone/core/singletons.py

```
@@ -31,8 +31,13 @@ def __new__(metacls, *args, **kwargs):
         return cls
 
     def _make_instance(cls, name, _create, *args, **kwargs):
+        media_type = kwargs.get("media_type", None)
```

Ruff (0.15.15)

[warning] Use `kwargs.get("media_type")` instead of `kwargs.get("media_type", None)`

Replace `kwargs.get("media_type", None)` with `kwargs.get("media_type")`

(SIM910)